examples/loader-canvas-to-png/src/data/us-map.png.js

```diff
@@ -0,0 +1,44 @@
+import * as canvas from 'canvas';
+import * as d3 from 'd3';
+import * as topojson from "topojson-client";
+
+// Get the map file from the US Atlas package
+// https://github.com/topojson/us-atlas
+const url = "https://cdn.jsdelivr.net/npm/us-atlas@3/counties-albers-10m.json";
+const us = await fetch(url).then(response => response.json());
+
+// Create and configure a canvas
+const width = 975;
+const height = 610;
+const cvs = canvas.createCanvas(width, height);
```

Comment:
```suggestion
const canvas = createCanvas(width, height);
```

Reply:
here I fear that this is creating an image in low resolution—maybe we should multiply everything by 2 for retina screens? And use context.scale(0.5, 0.5)